main/acle.md

```diff
@@ -5708,6 +5901,9 @@ bfloat types are only available when the `__bf16` type is defined, that is,
 when supported by the hardware. The bfloat types are all opaque types.  That is
 to say they can only be used by intrinsics.
 
+The FP8 types are all opaque types. That is to say they can only be used
+by intrinsics.
```

Comment:
That is to say they can only be used by intrinsics.

Change to: That is, they can only be used by intrinsics.